controllers/controller.js

```diff
@@ -48,4 +49,19 @@ res.status(200).send({ comments });
 }).catch((err) => {
 next(err)
 })
+}
+
+exports.postComment = (req, res, next) => {
+  console.log(req.params);
```

Comment:
remove logs before pushing

Reply:
Can't believe it, i was hunting for a log!!!